packages/api/src/functions/__tests__/graphql.test.ts

```diff
@@ -11,21 +27,19 @@ describe('global context handlers', () => {
     process.env.SAFE_GLOBAL_CONTEXT = '0'
   })
 
-  it('merges the apollo resolver and global context correctly', async () => {
+  it('merges the graphql-server resolver and global context correctly', async () => {
```

Comment:
I should no longer refer to graphql-server or Apollo -- they are simply resolvers.

```suggestion
  it('merges the  resolver and global context correctly', async () => {
```